proto/svc-storage-grpc-vertiport.proto

```diff
@@ -41,12 +41,10 @@ message Data {
     string name = 1;
     // additional description of the vertiport
     string description = 2;
-    // vertipad latitude location
-    double latitude = 3;
-    // vertipad longitude location
-    double longitude = 4;
+    // geographical area location of the vertiport
+    grpc.GeoPolygon geo_location = 3;
```

Comment:
This will be nice because we can use them as no-fly zones for itineraries where this port is not on their path